src/components/whiteboards/OnlineUsers.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <v-navigation-drawer
+    app
+    permanent
+    :expand-on-hover="true"
+    :mini-variant="true"
+    :clipped="$vuetify.breakpoint.lgAndUp"
+    :right="true"
+  >
+    <v-list>
+      <v-list-item class="px-2">
+        <v-list-item-avatar>
+          <v-img src="https://randomuser.me/api/portraits/women/85.jpg"></v-img>
+        </v-list-item-avatar>
+      </v-list-item>
+
+      <v-list-item link>
+        <v-list-item-content>
+          <v-list-item-title class="text-h6">
+            Sandra Adams
+          </v-list-item-title>
+          <v-list-item-subtitle>sandra_a88@gmail.com</v-list-item-subtitle>
+        </v-list-item-content>
+      </v-list-item>
+    </v-list>
+
+    <v-divider></v-divider>
+
+    <v-list
+      nav
+      dense
+    >
+      <v-list-item link>
+        <v-list-item-icon>
+          <v-icon>mdi-folder</v-icon>
+        </v-list-item-icon>
+        <v-list-item-title>My Files</v-list-item-title>
+      </v-list-item>
+      <v-list-item link>
+        <v-list-item-icon>
+          <v-icon>mdi-account-multiple</v-icon>
+        </v-list-item-icon>
+        <v-list-item-title>Shared with me</v-list-item-title>
+      </v-list-item>
+      <v-list-item link>
+        <v-list-item-icon>
+          <v-icon>mdi-star</v-icon>
+        </v-list-item-icon>
+        <v-list-item-title>Starred</v-list-item-title>
+      </v-list-item>
+    </v-list>
+  </v-navigation-drawer>
+</template>
+
+<script>
+export default {
+  name: 'Online'
```

Comment:
@johncrossman `name` 'Online' doesn't match the file name 'OnlineUsers' - intentional?

Reply:
Ah, not intentional. I'll fix.